### PR TITLE
ONEID-1323

### PR DESCRIPTION
**Pull request by DX-Bandwidth**
BYOIDP related documentation.
To view the site how it would look with the Identity stuff, un-comment all the places that have TODO ONEID-1304.

### CODE REVIEW

File: site/docs/identity/guides/settingUpIdP.mdx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- `usernameField` - This indicates which field in your SAML assertion represents the username of the user signing in. This value must match the username in our system.
+By default, this will be using the `NameId` of the `Subject` in the assertion.
+- `issuerUri` - this is the Entity ID of your SAML Identity Provider which will send the SAML assertion back to us upon a successful sign-in.
+- `singleSignOnUrl` - the SAML endpoint on your Identity Provider that we will `POST` to in order to initiate an authentication on behalf of a user.
+	- NOTE: we only support a `POST` binding when initiating SSO.
+- `signatureCertificate` - either a PEM or DER decoded certificate to verify the authenticity of your SAML Identity Provider.
+
+### Information you need from us
+
+Your identity provider will need information about our service provider to know whom to trust and whither to send information.
```
Comment (rstevens-bw) [inline site/docs/identity/guides/settingUpIdP.mdx:31]: whither -> whether

File: site/docs/identity/guides/settingUpIdP.mdx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+## FAQs
+### If I update an IdP to be active, what happens to other IdPs?
+
+<!-- TODO: this first sentence might not be true either. ONEID-1574 -->
+
+There can only be one active IdP per account. Therefore, if another IdP is already active when attempting to activate a different IdP, the initially active IdP will be set to inactive.
+
+### Can I delete an IdP while it is still active? 
+
+You cannot delete an IdP if it is currently active. You must first deactivate it before deleting the IdP. If you attempt to delete an IdP that is still active  you will recieve a `409 Conflict` HTTP status.
```
Comment (rstevens-bw) [inline site/docs/identity/guides/settingUpIdP.mdx:52]: nit pick, double space between `active` and `you`

**Review by rstevens-bw — DISMISSED** [review]

File: site/docs/identity/guides/settingUpIdP.mdx (hunk 1 above)
Comment (derelbenkoenig) [reply site/docs/identity/guides/settingUpIdP.mdx:31]: I meant "whither" in the sense of "to where"

Comment (derelbenkoenig) [reply site/docs/identity/guides/settingUpIdP.mdx:31]: People who don't know that word should simply go read some Tolkien

File: site/docs/identity/guides/settingUpIdP.mdx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  - bandwidth
+  - identity
+  - identity provider
+  - setup
+hide_title: true
+image: ../../static/img/bandwidth-logo.png
+---
+## Configuring Your Identity Provider
+
+Configuring your identity provider for Bandwidth will require you have a few pieces of information ready. You will need
```
[the remark this answers is not in the document]
Comment (derelbenkoenig) [reply site/docs/identity/guides/settingUpIdP.mdx]: oh yeah I forgot to do that

**Review by tmonck — DISMISSED** [review]

**Comment by DX-Bandwidth** [comment]
Corresponding Pull Request on [api-specs](https://github.com/Bandwidth/api-specs/pull/145) was merged.

**Review by derelbenkoenig — DISMISSED** [review]
I wrote a lot of this but I'll approve it anyway, we may still want more approvals from others but I'm indicating that in my opinion this is ready

**Review by rstevens-bw — DISMISSED** [review]

**Review by ajrice6713 — DISMISSED** [review]

**Review by ajrice6713 — APPROVED** [review]

**Review by derelbenkoenig — APPROVED** [review]

**Pull request merged**